task_reactredux/task1/src/containers/CounterAddDelContainer.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import CounterContainer from './CounterContainer';
+import AddDelCounter from '../views/AddDelCounter/index';
+
+class CounterAddDelContainer extends React.Component {
```

Comment:
I don't like the naming for this container. As for me, it sounds like this container should be responsible for `add` and `del`. I think a better name for this is something like `CountersParentContainer` or even better to use `CounterListContainer`